serverscripts/checkouts.py

```diff
@@ -386,6 +436,12 @@ def main():
             mode = "pipenv"
         elif os.path.exists(os.path.join(directory, "buildout.cfg")):
             mode = "buildout"
+        elif os.path.isfile(os.path.join(directory, ".venv/pyvenv.cfg")):
+            mode = "virtualenv"
+            bin_dir = os.path.abspath(os.path.join(directory, ".venv/bin"))
+        elif os.path.isfile(os.path.join(directory, "pyvenv.cfg")):
```

Comment:
Ik zag die `pyvenv.cfg` laatst ook verschijnen toen ik serverinfo naar virtualenv ombouwde, maar ik moest wel even kijken waar die nou voor diende.
a) Of ik heb niet goed opgelet en teveel dingen door elkaar gebruikt.
b) Of die pyvenv.cfg is misschien iets nieuws: dan zouden er oude deploys kunnen zijn waar deze nog niet bestaat. (Misschien niet erg, want de situatie gaat er in ieder geval niet op achteruit.)

Reply:
Je hebt gelijk, dit werkt niet met virtualenv < 20 (= alles voor 2020)